benchmark/python/control_flow/rnn.py

```diff
@@ -24,8 +24,8 @@
 from time import time
 
 import mxnet as mx
-import numpy as np
-from mxnet import gluon
+import numpy as onp
```

Comment:
Yes, 'o' in onp is 'official', which is used to distinguish between official numpy and MXNet numpy. Usually, user will do 
`from mxnet import np` and build their models with numpy operators from MXNet. This will provide numpy-compatible coding experience in MXNet for users who are familiar with official numpy.